style.css

```diff
@@ -230,4 +270,54 @@ cursor: pointer;
 .banner-content-desc{
     font-size: 1.8rem;
     line-height: 2.7rem;
+}
+
+.text-linethrough{
+    color: red;
+}
+
+/* footer */
+footer{
+    display: flex;
+    flex-direction: column;
+    background-color: #27474e;
+    background-color: var(--footer-bg-color);
+    color: #d4d9dd;
+}
+
+.footer-container{
+    display: flex;
+    justify-content: space-evenly;
+    font-size: 1.5rem;
+    flex-wrap: wrap;
+}
+
+.footer-container *{
+    flex-grow: 1;
+}
+
+.footer-container div{
+    padding: 1rem;
+}
+
+.footer-container ul{
+    padding: 1rem;
+}
+
+.list-body-item:hover{
+    transform: scale(1.01);
+    opacity: 1.1;
+    background-color: rgba( var(--footer-bg-color),0.8) ;
+}
+
+.footer-base{
+    display: flex;
+    justify-content: space-between;
+    padding: 1rem;
+    flex-wrap: wrap;
+}
+
+.footer-base-content{
+    display: flex;
+    justify-content: flex-end;
```

Comment:
Code looks clean easily readable. I just have few observations in UI, if you want to consider
![image](https://user-images.githubusercontent.com/82932172/154864987-d0236215-f747-4e9b-a65a-657db9299ce5.png)

The header/cards/hero-img look a little misaligned. Also, you may want to style the icons in nav. Seems they are having default a tag styling

Reply:
okay modified and update the tage